lang/src/test/kotlin/org/partiql/lang/eval/ExprValueTest.kt

```diff
@@ -0,0 +1,505 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+package org.partiql.lang.eval
+
+import com.amazon.ion.IonTimestamp
+import com.amazon.ion.IonValue
+import com.amazon.ion.Timestamp
+import com.amazon.ion.system.IonSystemBuilder
+import junitparams.JUnitParamsRunner
+import junitparams.Parameters
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.partiql.lang.ION
+import org.partiql.lang.errors.ErrorCode
+import org.partiql.lang.eval.time.Time
+import org.partiql.lang.util.isBag
+import org.partiql.lang.util.isMissing
+import org.partiql.lang.util.seal
+import java.math.BigDecimal
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.OffsetTime
+import java.time.ZoneOffset
+import kotlin.test.assertEquals
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+import kotlin.test.fail
+
+@RunWith(JUnitParamsRunner::class)
+class ExprValueTest {
+    companion object {
```

Comment:
It's sad Github does not detect this is a copy of ExprValueFactoryTest.kt and highlights the interesting changes.